### PR TITLE
Core/Misc: Fix static analysis issues

### PR DESCRIPTION
**Pull request by jackpoz**
**Changes proposed:**

-  Core/Misc: Fix static analysis issues
**Target branch(es):** 3.3.5/master

- [x] 3.3.5
- [ ] master

**Issues addressed:**

None

**Tests performed:**

None

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] Fix more issues

### CODE REVIEW

File: src/server/scripts/Commands/cs_debug.cpp [hunk 1]
```diff
@@ -274,44 +274,58 @@ class debug_commandscript : public CommandScript
...
             }
             parsedStream.put(commentToken[0]);
         }
         ifs.close();
 
-        uint32 opcode;
+        uint32 opcode = 0;
         parsedStream >> opcode;
 
+        if (opcode == 0)
```
Comment (jackpoz) [inline src/server/scripts/Commands/cs_debug.cpp:326]: Need to change this to !opcode

**Comment by illfated** [comment]
Looks quite ordered and easy to follow.

I assume real coders don't find it difficult to read, but for me it is somewhat hard to follow the logic in the change in `cs_debug.cpp`. I presume the logic stems from a Coverity entry, and is therefore needed to avoid issues.

**Comment by jackpoz** [comment]
all these fixes are from pvs-studio analysis at https://tcubuntu.northeurope.cloudapp.azure.com/pvsstudio/analysis/ . The cs_debug.cpp in particular is https://tcubuntu.northeurope.cloudapp.azure.com/pvsstudio/analysis/sources/cs_debug.cpp_16.html#ln273 , saying
>The 'ifs' stream is checked for EOF before reading from it, but is not checked after reading. Potential use of invalid data.

I was actually able to trigger an infinite loop in that code, now that's fixed

**Comment by illfated** [comment]
> I was actually able to trigger an infinite loop in that code, now that's fixed

Progress! 😄
